rotkehlchen/globaldb/handler.py

```diff
@@ -1180,6 +1181,76 @@ def get_historical_price_data(source: HistoricalPriceOracle) -> List[Dict[str, A
                  'to_timestamp': entry[3],
                  } for entry in query]
 
+    @staticmethod
+    def set_requests_cache_value(
```

Comment:
Actually not sure about the approach here. Maybe will be better to have methods set per cache type instead of the general-purpose methods? (but we can end up having too many methods)